lcr/__init__.py

```diff
@@ -48,43 +58,41 @@ def _login(self, user, password):
 
         # Enter the username
         login_input = WebDriverWait(self.driver, TIMEOUT).until(
-                        ec.presence_of_element_located(
-                            (By.CSS_SELECTOR, "input#okta-signin-username")
-                            )
-                        )
+            ec.presence_of_element_located(
+                (By.CSS_SELECTOR, "input#okta-signin-username")
+            )
+        )
         login_input.send_keys(user)
         login_input.submit()
 
         # Enter password
         password_input = WebDriverWait(self.driver, TIMEOUT).until(
-                ec.presence_of_element_located(
-                    (By.CSS_SELECTOR, "input.password-with-toggle")
-                    )
-                )
+            ec.presence_of_element_located(
+                (By.CSS_SELECTOR, "input.password-with-toggle")
+            )
+        )
         password_input.send_keys(password)
         password_input.submit()
 
-        # Wait until the page is loaded
         WebDriverWait(self.driver, TIMEOUT).until(
-                ec.presence_of_element_located(
-                    (By.CSS_SELECTOR, "input#memberLookupMain")
-                    )
-                )
+            ec.presence_of_element_located((By.CSS_SELECTOR, "input[type='search']"))
+        )
 
         # Get authState parameter.
         cookies = self.driver.get_cookies()
-        potential_cookie = [c for c in cookies if "ChurchSSO" in c['name']]
-        real_cookie = next(iter(potential_cookie))
-        churchcookie = real_cookie['value']
+        for c in cookies:
+            if "appSession" in c["name"]:
```

Comment:
instead of trying to figure out which cookies we need, just capture all of them.